Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -59,43 +65,63 @@ def main(argv):
 
 
 def init_ami_client(userkey='', usercert=''):
-    """ Initialisation of AMI client into the global variable
+    """ Establish a connection to AMI.
+
+    Initialize the global variable ami_client with the resulting
+    client object.
```

Comment:
Maybe add to this description `:raises:` section, as a clue to "under what circumstances it is expected to fail and how it will look"? I know we do not usually add this part, so "why here?"... but we had some discussions about "when to fail" (about this function and the similar one for Rucio, etc), and I remember myself extracting this summary from the code once or twice already, and I have a feeling it's not over yet.

Reply:
Done.
Note to self - if necessary, update #312 when this point is concluded: IIRC, there is at least one additional case for exception here.